src/ui/SearchField/styles.scss

```diff
@@ -122,6 +122,7 @@
       .search-result-table {
         width: 100%;
         border-collapse: collapse;
+        table-layout: fixed;
```

Comment:
Jeg har opplevd problemer med dette tidligere hvis f.eks ett felt står for majoriteten av innholdet og man skal ha en liten statuskolonne eller lignende ved siden av.

Men vi kjører igjennom og ser hva som skjer, hvis får heller løse det hvis det blir ett problem i fremtiden.

Reply:
👍